core/events/workspace_events.ts

```diff
@@ -57,15 +51,13 @@ export class FinishedLoading extends AbstractEvent {
    *
    * @returns JSON representation.
    */
-  override toJson(): AnyDuringMigration {
-    const json = {
-      'type': this.type,
-    };
+  override toJson(): FinishedLoadingJson {
+    const json = {'type': this.type} as FinishedLoadingJson;
```

Comment:
would `const json: FinishedLoadingJson = {'type': this.type}` allow you to avoid the cast?